2016/6xxx/CVE-2016-6555.json

```diff
@@ -1,18 +1,99 @@
 {
     "CVE_data_meta": {
-        "ASSIGNER": "cve@mitre.org",
+        "ASSIGNER": "cve@rapid7.com",
+        "DATE_PUBLIC": "2016-09-14T18:18:00.000Z",
```

Comment:
The blog shows "Nov 15, 2016"

Reply:
￼Indeed, but the PR was public earlier so I'm figuring that's the public date of the vuln. (That's the first time anyone knew about it who cared to look.)

Reply:
Okay, that makes sense!